schema.py

```diff
@@ -0,0 +1,118 @@
+from sqlalchemy import (Column, ForeignKey, DateTime, Boolean, Integer, Float, String)

```

Comment:
Two suggestions: 

1. PEP-8 suggests you break imports into up to three blocks, in this order: standard library; third party; local code. So: 
```
import datetime

import sqlalchemy

import db
```
2. This is totally up to you, but when I have long multiple import statements, I just think they look nice split up, like:
```
from sqlalchemy import (
    Column, 
    ForeignKey, 
    DateTime, 
    Boolean, 
    Integer, 
    Float, 
    String
)
```